src/main/java/edu/escuelaing/ieti/matchmaking/controllers/TeamController.java

```diff
@@ -18,15 +18,11 @@
 	@Autowired
     private TeamService teamService;
 	 @GetMapping
-	  public ResponseEntity<?> getAllTeams() {
+	  public ResponseEntity<?> getAllTeams() throws EntityNotFoundException {
```

Comment:
If the exception is meant to be propagated towards the frontend then maybe it shouldn't be a checked exception. You can inherit from RuntimeException and avoid the throws clause in every method. The @ExceptionHandler in the @ControllerAdvice would anyway take care of properly convert the error before sending it to frontend.